augur/merge.py

```diff
@@ -34,7 +38,21 @@
 future.  The SQLite 3 CLI, sqlite3, must be available.  If it's not on PATH (or
 you want to use a version different from what's on PATH), set the SQLITE3
 environment variable to path of the desired sqlite3 executable.
+
+**Sequences**
+
+Sequence files are merged in the order given. Naming files is optional. If
+names are provided, they will be checked against any named metadata for order
+and IDs.
+
+SeqKit is used behind the scenes to implement the merge, but, at least for now,
+this should be considered an implementation detail that may change in the
```

Comment:
Suggestion: don't hedge, just state.

```suggestion
SeqKit is used behind the scenes to implement the merge, but
this should be considered an implementation detail that may change in the
```